### DISCUSSION TITLE
[DONE] Simple Metronome - Type BPM

### DISCUSSION THREAD

**Discussion by shamlord** [yamadapc/augmented-audio#595] [Ideas]
The slider and tap UI in Simple Metronome is nice, but perhaps a user could set a beat by typing the number to make it easier to set an exact speed? My first impulse was to tap the number in the center of the UI.

For clarity, I need 16 BPM which I can't seem to do with the UI in its current form.

**Comment by yamadapc** [comment]
Let me try to fix this, that control is a text input. It's just disabled on the iOS version because I hadn't reviewed the keyboard mode.

**Comment by yamadapc** [comment]
Addressed on v1.9 which is up on the store for updating.
